bin.src/generateCoefficients.py

```diff
@@ -6,6 +6,7 @@
 from lsst.sims.movingObjects import Orbits
 from lsst.sims.movingObjects import ChebyFits
 
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Generate Chebyshev polynomial coefficients" +
```

Comment:
Another hack I introduced to address this problem: the code outputs coefficients at 30 day intervals so that, if I ask it to simulate 100 objects over the whole survey, I get 122 output files, each with all 100 objects simulated over one month.  That way, if the 100 objects can't complete in the wall time I asked for, I can at least save the months that did complete and restart the job from the first month that failed to complete.  This increases the number of output files.  I'm not worried about that, since we need to ingest into fatboy one month at a time anyway.